src/userlist/userListSelectors.js

```diff
@@ -0,0 +1,20 @@
+const statusOrder = (status) =>
```

Comment:
This could be more readable. Maybe something like the following?
```
const statusOrder = (status) =>
    case (status) {
        active:
            return 1;
        idle:
            return 2;
...
```